### PR TITLE
Nanoui misc changes

### PR DESCRIPTION
**Pull request by quardbreak**
DNM; Ports some nanoui changes from Eris:
* Force open optimization, window focus on open
* tgui css
* panic button to close all mob's uis

### CODE REVIEW

File: code/modules/nano/nanoui.dm [hunk 1]
```diff
@@ -432,11 +433,19 @@ nanoui is used to open and update nano browser uis
 		return // Will be closed by update_status().
 
 	show_browser(user, get_html(), "window=[window_id];[window_size][window_options]")
+	winset(user, window_id, "on-close=\"nanoclose \ref[src]\"")
 	winset(user, "mapwindow.map", "focus=true") // return keyboard focus to map
-	on_close_winset()
```
Comment (afterthought2) [inline code/modules/nano/nanoui.dm]: I believe doing it like this doesn't work consistently.

File: code/modules/nano/nanoui.dm [hunk 2]
```diff
@@ -432,11 +433,19 @@ nanoui is used to open and update nano browser uis
...
 	SSnano.ui_opened(src)
 
+ /**
+  * Move window up front.
+  *
+  * @return nothing
+  */
+/datum/nanoui/proc/focus()
+	winset(user, window_id, "focus=true")
+	winset(user, "mapwindow.map", "focus=true") // return keyboard focus to map
```
Comment (afterthought2) [inline code/modules/nano/nanoui.dm:448]: Testing this winset command in other contexts led me to the conclusion that it's inconsistent at best and does nothing at worst. Not opposed to it exactly, but I have doubts that this focuses reliably in an actual server deployment.

**Review by afterthought2 — COMMENTED** [review]
Isn't focusing the window as default kind of obnoxious (even assuming it works)?

File: code/modules/nano/nanoexternal.dm [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+  * Feel free to remove it once the bug is confirmed to be fixed.
+  *
+  * @return nothing
+  */
+/client/verb/resetnano()
+	set name = "Reset NanoUI"
+	set category = "OOC"
+
+	var/ui_amt = length(mob.open_uis)
+	for(var/datum/nanoui/ui in mob.open_uis)
```
Comment (PsiOmegaDelta) [inline code/modules/nano/nanoexternal.dm]: `close_user_uis(mob)`

**Pull request closed without merging**